md/bonita-container.md

```diff
@@ -0,0 +1,111 @@
+# Deploy a Bonita platform using docker
```

Comment:
Why naming the page bonita-container, instead of `Bonita docker installation` ? As this is the link of the page in the index